_component_design/selection-bucket.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Selection Bucket
+parent: Tables
+layout: component
+category: Components
+usage: >
+  A selection bucket contains lists of table values selected by the user. Each time a user selects a table value, a pill representing the selected value appears in the selection bucket. The selection bucket is located below the table and holds all selected values, across all table pages. The user can remove selected values from the selection bucket.
+preview-image: preview-images/selector-strip.svg
+status: stable
+resource: true
+last-modified: 2018-12-10
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a selection bucket when the table spans multiple pages. A selection bucket helps remind the user of prior selections, and provides a way for the user to easily deselect those values.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a selection bucket:
+
+- Only use a selection bucket when the table spans multiple pages.
+- By default, the selection bucket is collapsed.
```

Comment:
By default, the selection bucket can be collapsed or expanded by default, depending on the workflow. Default collapse provides more vertical space, but hides the selected values. Collapse provides the summary of the numbered selected.

Reply:
done.